packages/client/src/components/app/blocks/ChartBlock.svelte

```diff
@@ -85,7 +85,7 @@
           legend,
           animate,
           ...colors,
```

Comment:
In order for the custom colors to work, the array needs to be changed to an object.

```javascript
$: colors = c1 && c2 && c3 && c4 && c5 ? { c1, c2, c3, c4, c5 } : null
```

Reply:
I'm actually not sure what how the original implementation of this worked, but I changed it just pass `c1`, `c2` to the underlying chart component directly and it works now.